modules/deploy-app.adoc

```diff
@@ -20,15 +20,15 @@ From the {product-title} web console, you can deploy a test application from the
 ifndef::quickstart[]
 .Prerequisites
 
-* You logged in to {cluster-manager-url}.
-* You created an {product-title} cluster.
+* You logged in to the {hybrid-console-url}.
+* You created a {product-title} cluster.
 * You configured an identity provider for your cluster.
 * You added your user account to the configured identity provider.
 endif::[]
 
 .Procedure
 
-. From the {cluster-manager} {hybrid-console-second}, click *Open console*.
+. From the *Clusters* page of the {hybrid-console-second}, click *Open console*.
 
 . In the *Administrator* perspective, select *Home* -> *Projects* -> *Create Project*.
```

Comment:
Before this step, User will be redirected to openshift cluster console page and need to login with configured IDP credentials. Probably worth to mention the same step? Please review it.